tests/auto/rt_auto.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+set -eux
+
+export RT_COMPILER='intel'
+source ../detect_machine.sh
+echo "Machine ID: "+$MACHINE_ID
+export MACHINE_ID=$MACHINE_ID
+if [[ $MACHINE_ID = hera.* ]]; then
+  export PATH=/scratch1/NCEPDEV/nems/emc.nemspara/soft/miniconda3/bin:$PATH
+  export PYTHONPATH=/scratch1/NCEPDEV/nems/emc.nemspara/soft/miniconda3/lib/python3.8/site-packages
+  python rt_auto.py
```

Comment:
move `python rt_auto.py` at the end of the if-else block.
They all are `python rt_auto.py`